### DISCUSSION TITLE
How do I connect with OpenAI compatible APIs? (together.ai)

### DISCUSSION THREAD

**Discussion by jedgard25** [open-webui/open-webui#1828] [Q&A]
I have open webui container installed with docker. It runs fine. I go into settings and set the openAI api to `https://api.together.xyz/models` and enter my API key. It saves but when I search for models none appear. Am I doing this correctly?

If I restart the docker container, also, it doesn't save any of the information I've entered.

**Comment by alkeryn** [comment]
you need to save it in the .env file afaik, not sure you can add more than one though.

**Reply by justinh-rahb** [reply]
https://docs.openwebui.com/tutorial/openai

**Reply by alkeryn** [reply]
yea i figured it out, thanks.
turns out the name must also begin with openai/.

**Comment by rgaricano** [comment]
set it as openAI API compatible, with conn endpoint: `https://api.together.xyz/v1`
